app/templates/views/usage.html

```diff
@@ -39,13 +39,13 @@ <h2 class='heading-small'>Text messages</h2>
             {% endfor %}
           </div>
         </div>
-        <div class='govuk-grid-column-one-half pilot-disabled'>
+        <!-- <div class='govuk-grid-column-one-half pilot-disabled'>
```

Comment:
Since we're hiding this anyway, you could consider removing the `pilot-disabled` class. It may not be used elsewhere